packages/flutter_web_plugins/lib/src/plugin_registry.dart

```diff
@@ -25,6 +30,8 @@ class PluginRegistry {
   /// Registers this plugin handler with the engine, so that unrecognized
   /// platform messages are forwarded to the registry, where they can be
   /// correctly dispatched to one of the registered plugins.
+  ///
+  /// Only one [PluginRegistry] can be so registered at a time.
```

Comment:
nit: delete 'so'

Reply:
updated docs with more information while i was at it